.travis.yml

```diff
@@ -31,6 +31,7 @@ virtualenv:
   system_site_packages: true
 
 install:
+  - git clone https://github.com/akretion/server-tools.git ${HOME}/server-tools-base_exception -b 10.0-add-base_exception
```

Comment:
As the main PR is merge can you remove it https://github.com/OCA/server-tools/pull/775